btr-web/btr-main-app/components/individual-person/summary-table/row/CessationEntry.vue

```diff
@@ -0,0 +1,60 @@
+<script setup lang="ts">
```

Comment:
this component was just moved/renamed to be inside '/row'. Not sure why git didn't pick up on it just being a moved file (code is unchanged here)